server/__init__.py

```diff
@@ -26,3 +34,54 @@ async def convert_settings_overrides(
         return await super().convert_settings_overrides(
             source_version, overrides
         )
+
+    async def get_simple_actions(
+        self,
+        project_name: str | None = None,
+        variant: str = "production",
+    ) -> list[SimpleActionManifest]:
+        """Return a list of simple actions provided by the addon"""
+        output = []
+
+        # Add 'Create Project Folder Structure' action to folders.
+        output.append(
+            SimpleActionManifest(
+                identifier=f"{IDENTIFIER_PREFIX}.createprojectstructure",
+                label="Create Project Folder Structure",
+                icon={
+                    "type": "material-symbols",
+                    "name": "create_new_folder",
+                },
+                order=100,
+                entity_type="folder",
```

Comment:
This is an interesting caveat - we can't run "actions" currently in the web frontend on the **project level**. There is no UX for where actions may be displayed on project level. So there is the caveat here that you need to select an AYON folder entity to create the project structure... even though the folders on disk it actually creates have nothing to do with the selected AYON folder entity.

Slightly confusing - but I don't think we have anything better on the "web actions" frontend currently.